Source/Screens/ScreenBase.hpp

```diff
@@ -3,10 +3,18 @@
 
 #include <SFML/Graphics.hpp>
 
+const int SCREEN_BASE_NOT_CHANGING_SCREEN = -10;
```

Comment:
translate this variable name, pls

Reply:
I can't, lol. It's variable that denotes int value that returned when screen is not changed.

Reply:
why -10?
\

Reply:
-1 and 0 are already used